kraken-std/src/kraken/std/cargo/__init__.py

```diff
@@ -306,6 +306,7 @@ def cargo_build(
     name: str | None = None,
     project: Project | None = None,
     features: list[str] | None = None,
+    depends_on: Sequence[Task] | None = None,
```

Comment:
```suggestion
    depends_on: Sequence[Task] = (),
```

Then you don't need the `if` below.

Reply:
Can you provide a rationale in the changelog for this change?